src/main/java/Browser/BrowserFactory.java

```diff
@@ -0,0 +1,24 @@
+package Browser;
+
+import org.openqa.selenium.WebDriver;
+
+public enum BrowserFactory {
+    DEFAULT_BROWSER(new ChromeDriverBrowser()),
+    CHROME_DRIVER(new ChromeDriverBrowser()),
+    FIREFOX_DRIVER(new FirefoxBrowser()),
+    EDGE_BROWSER(new EgeBrowser());
+
+    BrowsersList list;
+
+    BrowserFactory(BrowsersList list) {
+      this.list = list;
+    }
+    WebDriver getDriver(){
```

Comment:
Add new line before getDriver. Methods should be separated. 
Also code formatting is incorrect here